dev-scheme/akku/akku-1.1.0-r100.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+GUILE_COMPAT=( 2-2 3-0 )
+inherit autotools guile
+
+DESCRIPTION="Language package manager for Scheme"
+HOMEPAGE="https://akkuscm.org/"
+
+if [[ "${PV}" == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/akkuscm/${PN}.git"
+else
+	SRC_URI="https://gitlab.com/akkuscm/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN}-v${PV}"
+fi
+
+LICENSE="GPL-3+"
+SLOT="0"
+
+REQUIRED_USE="${GUILE_REQUIRED_USE}"
+
+# tests require network access
+RESTRICT="test"
+
+RDEPEND="
+	${GUILE_DEPS}
+	net-misc/curl[ssl]
+"
+DEPEND="${RDEPEND}"
+
+# Installs into its own path
+# https://gitlab.com/akkuscm/akku/-/commit/d25da297fec2a2b16427359a2cbb0ec745dd8c58
+guile_set_common_vars() {
+	QA_PREBUILT="usr/$(get_libdir)/${PN}/guile/*/site-ccache/*"
+}
```

Comment:
hmm, it might be better just to set this globally rather than step over eclass functions